website/source/api/acl/auth-methods.html.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: api
+page_title: ACL Auth Methods - HTTP API
+sidebar_current: api-acl-auth-methods
+description: |-
+  The /acl/auth-method endpoints manage Consul's ACL Auth Methods.
+---
+
+-> **1.5.0+:**  The APIs are available in Consul versions 1.5.0 and later.
+
+# ACL Auth Method HTTP API
+
+The `/acl/auth-method` endpoints [create](#create-an-auth-method),
+[read](#read-an-auth-method), [update](#update-an-auth-method),
+[list](#list-auth-methods) and [delete](#delete-an-auth-method)
+ACL auth methods in Consul.  For more information about ACLs, please see
+the [ACL Guide](/docs/guides/acl.html).
+
+## Create an Auth Method
+
+This endpoint creates a new ACL auth method.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `PUT`  | `/acl/auth-method`           | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
```

Comment:
needs to be updated

Reply:
Where should the `#acls` anchor tag point now?